docs/source/modeling_with_aequilibrae/project_pieces/accessing_project_data.rst

```diff
@@ -0,0 +1,227 @@
+.. _accessing_project_data:
+
+Accessing Project Data
+======================
+
+An AequilibraE project helds geometric information that can be accessed by the user in 
+three different classes: ``Links``, ``Nodes``, and ``Zoning``. In this section, we'll
+cover the main points regarding them.
+
+``project.network.links``
+-------------------------
+
+This method allows you to access the API resources to manipulate the 'links' table.
+Each item in the 'links' table is a ``Link`` object.
+
+.. testsetup:: *
+
+    >>> folder = getfixture("create_path")
+
+.. code-block:: python
```

Comment:
The formatting of this part is pretty difficult to read. Is there a way to make it less dense?